gprofiler/profilers/java.py

```diff
@@ -258,7 +259,7 @@ def make_application_metadata(self, process: Process) -> Dict[str, Any]:
         # that loads other libs.
         libjvm_elfid = get_mapped_dso_elf_id(process, "/libjvm")
 
-        metadata = {"java_version": version, "libjvm_elfid": libjvm_elfid}
+        metadata = {"java_version": version, "libjvm_elfid": libjvm_elfid, "jvm_flags": get_jvm_flags(process)}
```

Comment:
I'm not sure this can handle nested dicts. Please check with the backend team if that's supported. If not, let's make it a JSON

Reply:
double checked with the backend team, its OK